ui/lib/core/addon/components/choose-pgp-key-form.hbs

```diff
@@ -27,6 +27,7 @@
         @textToCopy={{this.pgpKey}}
         @color="secondary"
         @onError={{(fn (set-flash-message "Clipboard copy failed. The Clipboard API requires a secure context." "danger"))}}
+        @isTruncated={{true}}
```

Comment:
I think adding this arg makes things less accessible - is it necessary?

Reply:
Yes, this was sort of the point of the original -- letting it overflow means the copy button is only clickable if you scroll. The data shown is also something the user already has (they uploaded it) so the contents are less than critical IMO